polaris-icons/icons/ReceivablesIcon.yml

```diff
@@ -0,0 +1,10 @@
+name: Receivables
+description: Used to direct merchants to accounts receivables
+keywords:
+  - finances
+  - financial accounts
+  - transactions
+authors:
+  - Eduardo Mujica
+date_added: 2024-06-07
+date_modified: 2024-06-07
```

Comment:
The icon doesn't speak to `recievables` it is a down arrow with a line. Could we use an existing icon or rename this to be more usable across the Admin?